src/main/java/brandkon/ProductService.java

```diff
@@ -0,0 +1,65 @@
+package brandkon;
+
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+public class ProductService {
+
+    private ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    //상품 목록 조회
+    public List<ProductResponse> getAllProducts(){
+        List<ProductResponse> responses = productRepository.findAll()
+                .stream()
+                .map(product ->
+                        new ProductResponse(product.getId(), product.getBrand().getName() ,product.getName(), product.getPrice(),""))
+                .toList();
+        return responses;
```

Comment:
참고로 변수를 거치지 않고 한 번에 바로 return해도 됩니다
```suggestion
        return productRepository.findAll()
                .stream()
                .map(product ->
                        new ProductResponse(product.getId(), product.getBrand().getName() ,product.getName(), product.getPrice(),""))
                .toList();
```